src/App.tsx

```diff
@@ -1,14 +1,37 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
+import { useDispatch, useSelector } from 'react-redux';
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
+import { RootState } from './app/store';
+import { getTodos } from './api';
+import { actions as initialActions } from './features/initialTodos';
+import { actions as todosActions } from './features/todos';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
 export const App: React.FC = () => {
+  const dispatch = useDispatch();
+  const initialStore = useSelector((state: RootState) => state.initialTodos);
+  const filteredStore = useSelector((state: RootState) => state.todos);
```

Comment:
you can create your own hook and do not add the type for state everywhere

look [here](https://redux-toolkit.js.org/tutorials/typescript)